hasjob/templates/macros.html.jinja2

```diff
@@ -207,3 +223,17 @@
     </script>
   {%- endwith %}
 {%- endmacro -%}
+
+{%- macro funnel_stat_setup(max_counts, paginated) -%}
+  <script type="text/javascript">
+    $(function(){
+      {%- if not paginated -%}
+        window.Hasjob.FunnelStat.createGradientColourBar('impressions', {{max_counts['max_impressions']}});
+        window.Hasjob.FunnelStat.createGradientColourBar('views', {{max_counts['max_views']}});
+        window.Hasjob.FunnelStat.createGradientColourBar('opens', {{max_counts['max_opens']}});
+        window.Hasjob.FunnelStat.createGradientColourBar('applied', {{max_counts['max_applied']}});
+      {%- endif %}
+      window.Hasjob.FunnelStat.updateFunnel();
+    });
+  </script>
+{%- endmacro -%}
```

Comment:
This macro will insert a `<script>` tag in the middle of an `<ol>` of stickies. This will impact the spacing of stickies, as the stickie area uses `inline-block` -- there will be one space inserted whenever the batch loads.

Since you only need to call `window.Hasjob.FunnelStat.updateFunnel()`, just do this in the success condition of the loader. Don't send it from server-side.